.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
   "editor.codeActionsOnSave": {
     "source.organizeImports": "always"
-  }
+  },
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "typescript.preferences.importModuleSpecifier": "non-relative"
```

Comment:
Favor full path imports for clarity, e.g. `@/app/prt-staking/constants`

Reply:
The only thing I'd be wondering here is if then imports are not just one big pile - as organizeImports would all put it in the same block?

Reply:
Looks like it still keeps them separate :+1: 

https://github.com/IndexCoop/index-app/pull/1384/files#diff-8cc522524edba5381be22a9ad93761e10b46d58072beffcc7ee251b5ababf8c7R1-R5